src/utils/media-utils.js

```diff
@@ -22,6 +22,8 @@ function b64EncodeUnicode(str) {
 }
 
 export const proxiedUrlFor = (url, index) => {
+  if (url.startsWith("webrtc://")) return url;
```

Comment:
We should probably more appropriately check for `!(http || https)` since that's all it makes sense to send farspark (The experiment I did with webtorrent also required a similar change to skip `magnet://` links)